src/List.js

```diff
@@ -1,31 +1,53 @@
-import React from 'react';
-import { FirestoreCollection } from 'react-firestore';
+import React, { useContext } from 'react';
+import firebase from 'firebase';
 import NavLinks from './NavLinks';
-import useTokenHook from './useTokenHook';
+import { ItemsContext } from './ItemsContext';
 
 const List = () => {
-  const { token } = useTokenHook();
+  const items = useContext(ItemsContext);
+
+  const now = new Date(Date.now());
+
+  const handleChange = async e => {
+    const newData = { lastPurchased: Date.now() };
+    const db = firebase.firestore();
+
+    const itemRef = db.collection('items').doc(e.target.id);
```

Comment:
This is probably nit-picking but I would like to know the reason why you decided to name this variable `itemRef`. Typically in React, the `somethingRef` naming convention is used to name the variable that contains what's returned by the `useRef` hook (for example: `const somethingRef = useRef()`) so I would consider sticking to `item` as a variable name so that other developers don't assume it's a reference as defined in React.

Reply:
@yvesgurcan is this a pattern clash of some kind? Using `ref` in variable names like `const databaseRef = firebase.firestore().collection('items');` is pretty common in Firestore, isn't it?